src/Service/StorageApiService.php

```diff
@@ -32,7 +32,7 @@ public function __construct(RequestStack $requestStack, $storageApiUrl = 'https:
     /**
      * @return \Closure
      */
-    public static function getStepPollDelayFunction()
+    public function getStepPollDelayFunction()
```

Comment:
Preco nenechat static, ked je to "pure" funkcia, nepouziva ziadne props triedy? 
Nie je to zbytocny BC break?

Reply:
jo vidis to, to jsem musel udelat, protoze si to stezovalo v testech, ze nemuze na mock objektu volat staticka funkce (driv se nevolala, ted se vola kvuli inicializaci branch klienta). Tak jsem to zkusil takhle, abych to nemusel cely predelavat

Reply:
Aha, tak potom ok.